.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -60,8 +66,11 @@ This change added tests and can be verified as follows:
 - [ ] The binary protocol
 - [ ] The REST endpoints
 - [ ] The admin CLI options
+- [ ] The metrics
 - [ ] Anything that affects deployment
 
+<!-- NOTE: A PIP is required if the PR will change any parts of the above. See https://github.com/apache/pulsar/blob/master/wiki/proposals/PIP.md -->
```

Comment:
if fix schema bugs, don't need the PIP, I think the schema option can be removed from here. PRs related to the schema, if you need PIP it's already included in `The default values of configurations` `The public API` etc.